public/app/features/dashboard/components/GenAI/GenAIPanelTitleButton.tsx

```diff
@@ -19,9 +19,16 @@ const TITLE_GENERATION_STANDARD_PROMPT =
 
 export const GenAIPanelTitleButton = ({ onGenerate, panel }: GenAIPanelTitleButtonProps) => {
   const messages = React.useMemo(() => getMessages(panel), [panel]);
-  const onClick = React.useCallback(() => reportGenerateAIButtonClicked(EventSource.panelTitle), []);
 
-  return <GenAIButton messages={messages} onClick={onClick} onGenerate={onGenerate} loadingText={'Generating title'} />;
+  return (
+    <GenAIButton
+      messages={messages}
+      onGenerate={onGenerate}
+      loadingText={'Generating title'}
+      toggleTipTitle={'Improving your panel title'}
```

Comment:
update tense to feel more natural
```suggestion
      toggleTipTitle={'Improve your panel title'}
```